news-app/src/API.js

```diff
@@ -8,13 +8,25 @@ const myApi = axios.create({
     return myApi.get(`/users/${username}`).then(({data}) => {
         const {user} = data;
       return user;
+    }).catch((err)=>{
+      return <h2>An error occurred: {err} </h2>
     });
   };
 
+  export const fetchUsers = (username) => {
+    return myApi.get(`/users`).then(({data}) => {
+        const {users} = data;
+      return users;
+    })
+  };
+
+
   export const fetchTopics = () => {
     return myApi.get(`/Topics`).then(({data}) => {
         const {topics} = data;
       return topics;
+    }).catch((err)=>{
+      return <h2>An error occurred: {err} </h2>
```

Comment:
Catching errors here should be a bit redundant as they should be caught in your backend
You can't return JSX here, only in the components